server/src/lib/config.rs

```diff
@@ -47,6 +52,14 @@ impl Config {
     pub fn devs(&self) -> IterDevs {
         IterDevs { inner: self.devices.iter() }
     }
+
+    pub fn block_dev<R: propolis::block::BlockReq>(
+        &self,
+        name: &str,
+    ) -> Arc<dyn propolis::block::BlockDev<R>> {
+        let entry = self.block_devs.get(name).unwrap();
```

Comment:
I'm willing to allow slightly more tolerance in the CLI tools for unwrap - because they're for testing anyway - but for anything in the server, I'd really like to be cautious about unwraps. 

Both here and in `BlockDevice::block_dev()` below, there are many conditions in which we could unwrap and panic - can we propagate the result upwards instead?

Reply:
agreed, fixed in 9036f7a